recipes/logr/all/conanfile.py

```diff
@@ -34,7 +36,7 @@ def requirements(self):
             fmt_ref = "fmt/9.1.0"
             spdlog_ref = "spdlog/1.11.0"
         else:
-            fmt_ref = "fmt/8.1.1"
+            fmt_ref = "fmt/8.0.1"
```

Comment:
Why going back in ``fmt`` version?

Reply:
Because there is a version conflict between fmt/8.1.1 and spdlog/1.9.0.

https://github.com/conan-io/conan-center-index/blob/0580eeb5763a18b4c7034b5092ad864dabc23ba2/recipes/spdlog/all/conanfile.py#L56-L63

I updated spdlog/1.10.0 to upgrade fmt to 8.1.1.